javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,31 @@
+var text = 'hi world';
+var $tutor = document.querySelector('.tutor');
+var characters = text.split('');
+
+
+function createSpan() {
+  for (var i = 0; i < text.length; i++) {
+    var $span = document.createElement('span');
+    $span.textContent = text[i];
+    $tutor.append($span);
+
+  }
+}
+
+createSpan();
+var span = document.querySelectorAll('span');
+var firstSpan = document.querySelector('span');
+firstSpan.className = 'cursor';
+
+function matchKeys ({ key }) {
+  for (var j = 0; j < span.length; j++) {
```

Comment:
You don't need a loop for this assignment.

Reply:
![Screen Shot 2021-03-12 at 11 18 46 AM](https://user-images.githubusercontent.com/74885918/110987984-c90bff00-8324-11eb-8542-54a624463907.png)